lib/stripe/checkout/session.ex

```diff
@@ -241,6 +246,7 @@ defmodule Stripe.Session do
           optional(:payment_intent_data) => payment_intent_data,
           optional(:payment_method_options) => payment_method_options(),
           optional(:setup_intent_data) => setup_intent_data(),
+          optional(:billing_address_collection) => billing_address_collection(),
```

Comment:
Can we simplify and inline `String.t()`?  What do you think?

Reply:
Yea no problem, just following some of the other patterns.